scripts/config/cassandra.toml

```diff
@@ -0,0 +1,36 @@
+[cassandra-store]
+type = "cassandra"
+create_db_template = """
+CREATE KEYSPACE IF NOT EXISTS %s WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1}  AND durable_writes = true
+CREATE TABLE IF NOT EXISTS %s.metric_idx (
+    id text,
+    orgid int,
+    partition int,
+    name text,
+    metric text,
+    interval int,
+    unit text,
+    mtype text,
+    tags set<text>,
+    lastupdate int,
+    PRIMARY KEY (partition, id)
+) WITH compaction = {'class': 'SizeTieredCompactionStrategy'}
+    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}
+"""
+
+[cassandra-idx]
+type = "scylladb"
```

Comment:
what does this type do, why is it needed and why is it set to scylladb in the cassandra file?

Reply:
nothing special, it was informative. Moreover the value here is an error during rebase i'll fix this